script/elastic/up.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# https://hub.docker.com/_/elasticsearch/
+
+
+sysctl -w vm.max_map_count=262144
```

Comment:
The ideal situation is check max_map_count size before ask sudo to change it just `cat /proc/sys/vm/max_map_count` will give you the value. though some transformation need to be made in shell. Anyway since *nix won't ask pwd for sudo in same shell, it's not a big deal.